lib/engine/game/g_1880/game.rb

```diff
@@ -13,6 +13,7 @@ class Game < Game::Base
         include_meta(G1880::Meta)
         include Map
         include Entities
+        include StubsAreRestricted
```

Comment:
Do you also need to `require_relative '../stubs_are_restricted'` before including this module?

Reply:
yep, added